tests/components/nest/test_media_source.py

```diff
@@ -692,3 +720,314 @@ async def test_multiple_devices(hass, auth, hass_client):
         hass, f"{const.URI_SCHEME}{DOMAIN}/{device2.id}"
     )
     assert len(browse.children) == 3
+
+
+@pytest.fixture
+def event_store() -> Generator[None, None, None]:
+    """Persist changes to event store immediately."""
+    m = Mock(spec=float)
+    m.return_value = 0
+    with patch(
+        "homeassistant.components.nest.media_source.STORAGE_SAVE_DELAY_SECONDS",
+        new_callable=m,
```

Comment:
`new=0` is enough I think.

Reply:
Thank you, i thought i had to be missing something here. Sent https://github.com/home-assistant/core/pull/62336 to address.